### ISSUE TITLE
Feature request: mirror manual modifications / transformations with Python code

### ISSUE THREAD

**Issue by aivarannamaa** [dmnfarrell/pandastable#60]
It would be nice if dataexplore had a Python output pane which shows corresponding Python code for each manual action on data. 

This would allow the user to 

- learn pandas / mpl 
- retrieve a log of his/her actions

**Comment by dmnfarrell** [comment]
Good idea, though it would not be that trivial to implement probably. I'm currently thinking of rewriting this application using pyqt so not sure about adding features at the moment. I'll put it on the to do list!
